src/app/components/team/Similarity/SimilarityComponent.js

```diff
@@ -80,9 +99,12 @@ const SimilarityComponent = ({
 
   const handleVectorModelToggle = (useVectorModel) => {
     if (!useVectorModel) {
-      handleSettingsChange('text_similarity_model', ELASTICSEARCH_MODEL);
+      const newSettings = { ...settings };
+      newSettings.alegre_model_in_use = ELASTICSEARCH_MODEL;
+      newSettings.text_similarity_model = ELASTICSEARCH_MODEL;
```

Comment:
I've added this line. This goes beyond the UX work in this ticket, but it seems there was some miscommunication between front-end and back-end on the `alegre_model_in_use` variable. The backend assumes it is the full list of models to use while the front end treated it as only the vector models. As a result we have some queries where elasticsearch is not being used to query despite the UX making seem it would be (there is no way to disable Elasticsearch in the UX, which is fine for now). Throughout the UX we should treat `alegre_model_in_use` and `text_similarity_model` as identical. They should always be the same. It was a historical mistake to have two variables and eventually we should remove one of them, but for now we'll just treat them the same throughout.